pkg/config/config.go

```diff
@@ -37,6 +37,19 @@ type ProjectConfig struct {
 	APISecret string `yaml:"api_secret"`
 }
 
+type Project struct {
+	Subdomain string
+}
+
+type AccessKey struct {
```

Comment:
what do these do?

Reply:
These are temporary types we deserialize into before we write project config to our config file. We don't have access to `backend-common` here due to OSS, so basically just recreating a minimal struct that has only the parts we need.